cellbase-lib/src/main/java/org/opencb/cellbase/lib/builders/clinical/variant/GwasIndexer.java

```diff
@@ -270,21 +262,27 @@ private void processGwasCatalogLine(String[] values, TabixReader dbsnpTabixReade
 
                     // Scores management
                     GwasAssociationStudyTraitScores scores = new GwasAssociationStudyTraitScores();
-                    try {
-                        scores.setPValue(Double.parseDouble(values[27]));
-                    } catch (NumberFormatException e) {
-//                        logger.warn(e.getMessage() + ". Parsing pValue: " + values[27]);
+                    if (StringUtils.isNotEmpty(values[27])) {
+                        try {
+                            scores.setPValue(Double.parseDouble(values[27]));
+                        } catch (NumberFormatException e) {
+                            logger.warn(e.getMessage() + ". Parsing pValue: " + values[27]);
+                        }
                     }
-                    try {
-                        scores.setPValueMlog(Double.parseDouble(values[28]));
-                    } catch (NumberFormatException e) {
-//                        logger.warn(e.getMessage() + ". Parsing pValue mlog: " + values[28]);
+                    if (StringUtils.isNotEmpty(values[28])) {
+                        try {
+                            scores.setPValueMlog(Double.parseDouble(values[28]));
+                        } catch (NumberFormatException e) {
+                            logger.warn(e.getMessage() + ". Parsing pValue mlog: " + values[28]);
```

Comment:
I understand that this exception will happen be too ofter to print the whole stacktrace, which would not add much information

Reply:
Exact.